### ISSUE TITLE
Investigate Paging Strategies for Improved Performance

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#6038]
# Description
Continuing investigation work for Solr responses and paging to research real world load and performance issue mitigation.

# Done Looks Like

**Comment by joncameron** [comment]
Correlation can be found; causation would be a lot harder, especially with our current tools.

**Comment by elynema** [comment]
@joncameron From my notes, I thought we would update this ticket to block deep pagination and present a nice info page to any user about what is happening. We would need to define how many pages / results represent deep pagination, both for facets and for results. We could also include a done looks like of pulling a day's worth (or more) of deep pagination from the logs for scanning by humans so we are comfortable that it looks like bot activity and not humans.

Does this match your take-away from the conversation? We could also run this policy of blocking deep pagination past the MCO Working Group to see if there are any concerns.

**Comment by joncameron** [comment]
Yes, that sounds right. I'll update the text in this issue to reflect that.
